### ISSUE TITLE
Inconsistency between live paper trader and backtest.

### ISSUE THREAD

**Issue by rumrush** [askmike/gekko#1728]
* **I'm submitting a ...**
[ X ] bug report
[ ] feature request
[ ] question about the decisions made in the repository

* **Action taken** (what you did)
I've perform a live paper trade for a couple of hours with my bot and compared the result with a backtest paper trade on exactly the same data that the live paper trader received.

* **Expected result** (what you hoped would happen)
When fed with exactly the same data, the paper trader should output the same result when in a live session and in a backtest.

* **Actual result** (unexpected outcome)
The trades are not performed at the same time and the live paper trader did a lot more trades compared to the backtest which makes the backtest irrelevant if the strat don't act the same in the backtest and in live paper trade.

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, etc)

Backtest result:
![screen_shot_2018-01-19_at_8 11 51_am](https://user-images.githubusercontent.com/11467733/35153297-f2972990-fcf3-11e7-90c8-d531ba8fd146.png)

Live paper trader result:
![screen_shot_2018-01-19_at_8 09 26_am](https://user-images.githubusercontent.com/11467733/35153278-dad12676-fcf3-11e7-9688-84c569697417.png)

**Comment by gvdhoven** [comment]
Was the config also exactly thesame? I found out that the fees and slippage can be different between live/paper trading.

**Comment by rumrush** [comment]
Yes 0.1 fees and 0.05 slippage and my strategy doesn't expose any configs it's all hardcoded so the config is exactly the same. I also tested it on Binance and Bitfinex with similar results. Also for more info, I was running the strategy on 1m candles.

**Comment by racineda** [comment]
I am also seeing this. Backtesting does not match my live trader. Just last night my live trader should have sold as reported by the backtest but there is no record of it attempting a sell at the time the backtest reported a sell. Using GDAX USD/LTC pair. Just updated to the latest in develop (7dfdc9927e44e7be9f758fada5694caf3706393c) and reinstalled.

**Comment by ipansensoy** [comment]
Also experienced this backtest and live is so different we ended up flipping the buy sell logic on live

**Comment by rumrush** [comment]
I've found that the issue seems to come from the use of multiples indicators of Talib and Tulip library in backtest.

If your are using those, the only way I found to get the right result was to rewrite all the indicators that I was using natively in Gekko.

For exemple, in my tests, I was using a MACD with an Awesome Oscillator indicator from Talib library and the results were extremely good in the backtest, but they were really bad in live paper trade. So I decided to write my own Awesome Oscillator and use Gekko's MACD and now the results were similar in backtest and in live paper trade.

So from now on, I never use Talib or Tulip indicators anymore until there is a fix for this issue.

Btw, Talib and Tulip results seem to be good when you use only one indicators in your strategy.
